src/nvim/api/win_config.c

```diff
@@ -129,7 +129,12 @@
 ///            fractional.
 ///   - focusable: Enable focus by user actions (wincmds, mouse events).
 ///       Defaults to true. Non-focusable windows can be entered by
-///       |nvim_set_current_win()|.
+///       |nvim_set_current_win()|, or, when the `mouse` field is set to true,
+///       by mouse events.
+///   - mouse: Specify how this window affects mouse events.
```

Comment:
```suggestion
///   - mouse: Specify how this window treats mouse events.
```

Reply:
No

Reply:
That's a bit pedantic though. Being technically correct leads to awkward documentation without any benefit AFAICT.